web-ui/src/streaming/PipelineTable.tsx

```diff
@@ -0,0 +1,668 @@
+                  )
+                }
+              ]}
+              rows={inputs}
+              sx={{ flex: 1 }}
+              hideFooter
+            />
+          </Paper>
+        </Grid>
+
+        <Grid item xs={12}>
+          <Paper>
+            <DataGridPro
+              autoHeight
+              getRowId={(row: ConnectorData) => row.ac.uuid}
+              columns={[
+                {
+                  field: 'name',
+                  headerName: 'Output',
+                  flex: 0.5,
+                  valueGetter: params => params.row.c?.name || 'unknown'
+                },
+                {
+                  field: 'config',
+                  valueGetter: params => params.row.ac.config,
+                  headerName: 'From View',
+                  flex: 0.8
+                },
+                {
+                  field: 'records',
+                  headerName: 'Records',
+                  flex: 0.15,
+                  renderCell: params =>
+                    format('.1s')(outputMetrics?.get(params.row.ac.config.trim())?.transmitted_records || 0)
+                },
+                {
+                  field: 'traffic',
+                  headerName: 'Traffic',
+                  flex: 0.15,
+                  renderCell: params =>
+                    humanSize(outputMetrics?.get(params.row.ac.config.trim())?.transmitted_bytes || 0)
+                },
+                //{
+                //  field: 'enabled',
+                //  headerName: 'Active',
+                //  flex: 0.15,
+                //  renderCell: () => <Switch defaultChecked disabled />
+                //},
+                {
+                  field: 'action',
+                  headerName: 'Action',
+                  flex: 0.15,
+                  renderCell: params => (
+                    <>
+                      <Tooltip title='Inspect Connector'>
+                        <IconButton
+                          size='small'
+                          onClick={() =>
+                            router.push('/streaming/introspection/' + props.row.config_id + '/' + params.row.ac.config)
+                          }
+                        >
+                          <Icon icon='bx:show' fontSize={20} />
+                        </IconButton>
+                      </Tooltip>
+                      <Tooltip title='Remove Connector from the Pipeline'>
+                        <IconButton size='small' href='#'>
+                          <Icon icon='bx:trash-alt' fontSize={20} />
+                        </IconButton>
+                      </Tooltip>
+                    </>
+                  )
+                }
+              ]}
+              rows={outputs}
+              sx={{ flex: 1 }}
+              hideFooter
+            />
+          </Paper>
+        </Grid>
+      </Grid>
+    </Box>
+  ) : (
+    <Box>Loading...</Box>
+  )
+}
+
+// TODO: We currently can't get the pipeline status from the manager, so we're
```

Comment:
have you filed a bug for this?

Reply:
fixed with https://github.com/vmware/database-stream-processor/issues/357